docs/source/en/glossary.mdx

```diff
@@ -199,36 +229,36 @@ because this is the way a [`BertModel`] is going to expect its inputs.
 
 ## L
 
-### Labels
+### labels
 
 The labels are an optional argument which can be passed in order for the model to compute the loss itself. These labels
 should be the expected prediction of the model: it will use the standard loss in order to compute the loss between its
 predictions and the expected value (the label).
 
 These labels are different according to the model head, for example:
 
-- For sequence classification models (e.g., [`BertForSequenceClassification`]), the model expects a tensor of dimension
+- For sequence classification models ([`BertForSequenceClassification`]), the model expects a tensor of dimension
   `(batch_size)` with each value of the batch corresponding to the expected label of the entire sequence.
-- For token classification models (e.g., [`BertForTokenClassification`]), the model expects a tensor of dimension
+- For token classification models ([`BertForTokenClassification`]), the model expects a tensor of dimension
   `(batch_size, seq_length)` with each value corresponding to the expected label of each individual token.
-- For masked language modeling (e.g., [`BertForMaskedLM`]), the model expects a tensor of dimension `(batch_size,
+- For masked language modeling ([`BertForMaskedLM`]), the model expects a tensor of dimension `(batch_size,
   seq_length)` with each value corresponding to the expected label of each individual token: the labels being the token
   ID for the masked token, and values to be ignored for the rest (usually -100).
-- For sequence to sequence tasks,(e.g., [`BartForConditionalGeneration`], [`MBartForConditionalGeneration`]), the model
+- For sequence to sequence tasks,([`BartForConditionalGeneration`], [`MBartForConditionalGeneration`]), the model
   expects a tensor of dimension `(batch_size, tgt_seq_length)` with each value corresponding to the target sequences
-  associated with each input sequence. During training, both *BART* and *T5* will make the appropriate
-  *decoder_input_ids* and decoder attention masks internally. They usually do not need to be supplied. This does not
+  associated with each input sequence. During training, both BART and T5 will make the appropriate
+  `decoder_input_ids` and decoder attention masks internally. They usually do not need to be supplied. This does not
```

Comment:
Would be great to expand this section for other modalities as well (can be a followup PR!). And also leave one last remark telling the user to always check the model documentation to be sure what the label are called.

Reply:
Great suggestion, I'll add it to my list!